client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -51,12 +50,24 @@
 		}
 	}
 
+	.design-selector__design-option {
+		transition: transform 0.3s ease-out, box-shadow 0.3s ease-out;
+		box-shadow: rgba( 0, 0, 0, 0.2 ) 0 0 0;
+		cursor: pointer;
+
+		&:focus,
+		&:hover {
+			transform: scale( 1.03 );
+			box-shadow: rgba( 0, 0, 0, 0.2 ) 0 0 15px;
+		}
+	}
+
 	.design-selector__option-name {
+		@include onboarding-font-recoleta;
 		margin-top: 16px;
 		display: block;
 		text-align: center;
 		width: 100%;
-		font-size: 24px;
-		@include onboarding-font-recoleta;
+		font-size: 18px;
```

Comment:
This change is font-size design tweak requested in https://github.com/Automattic/wp-calypso/issues/40394

Reply:
Let's get the font size+removing recoleta in separately, @alshakero